### PR TITLE
Add in unit testing framework (closes #9 closes #2)

### LINKED ISSUE #2: Unit testing framework
At the appropriate moment (soon?), start and/or document a unit testing framework; see BIDS and HED for models. 

Note: MKS is not clear whether these unit tests include, or are separate from, the validation over the reference datasets at https://github.com/bids-standard/bids-examples and https://github.com/hed-standard/hed-examples. Leaving as a single issue for now. 

(Note also: https://github.com/psych-ds/example-datasets currently contains contributed datasets that do not meet the spec, and this is an expected state at this point.  In the future, will need to establish pathways for how people can contribute dataset examples (a) generally and (b) for unit testing.)

### LINKED ISSUE #9: Create a canonical set of example datasets for testing
# TL;DR
The repo should include a directory of variously valid/invalid datasets to run through whenever the main.test.ts unit test is called.

# Detailed View
It must include information about what validator results are expected to be received, in order for testing to work

### PR DESCRIPTION
**Pull request by bleonar5**
closes #9 and closes #2

### CODE REVIEW

**Review by okaycj — CHANGES_REQUESTED** [review]
Could you add instructions into the readme on how to run the testing suite?

**Review by becky-gilbert — APPROVED** [review]
👏 👏 👏 
I agree with CJ that it would be great to see some more info in the Readme about running tests.
